File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        HashSet<Integer> detectDuplicates = new HashSet<Integer>();
        for (int i : nums) {
            if (detectDuplicates.contains(i)) {
                return true;
            }
            detectDuplicates.add(i);
        }
        return false;
    }
}

File: Data Structures & Algorithms/remove-element/submission-0.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        int l = 0;
        int r = nums.Length;

        while (l < r) {
            if (nums[l] == val) {
                nums[l] = nums[--r];
            }
            else {
                l++;
            }
        }
        return r;
    }
}